tests/unit/lms/services/canvas_api/_authenticated_test.py

```diff
@@ -72,7 +72,7 @@ def test_send_raises_CanvasAPIAccessTokenError_if_it_cannot_refresh(
             )
 
     def test_get_token(
-        self, authenticated_client, basic_client, token_store, token_response
+        self, authenticated_client, basic_client, token_store_service, token_response
```

Comment:
Lots of renaming the `token_store` fixture to `token_store_service` in tests. But `token_store_service` was consistent with how our other service fixtures are named